docker-compose.yml

```diff
@@ -45,7 +45,7 @@ services:
 
   childchain:
     image: elixir-omg:dockercompose
-    entrypoint: /bin/bash -c "./launcher.py && mix xomg.child_chain.start --config ~/config.exs"
+    entrypoint: /bin/bash -c "./launcher.py && elixir --erl '-sname childchain' -S mix xomg.child_chain.start --config ~/config.exs"
```

Comment:
maybe give a hint about how to do this for the non-docker setup (in `docs/manual_....md`)? Also possibly a hint on how to use this in general in `README.md`

Reply:
It's a bit difficult at the moment, because they all presume you boot into the app in an interactive way anyhow (`iex`).

Reply:
It might be better for us to wait and rewrite these sections once we get distillery out? And then write these for developers (`how to develop`, and `how to deploy`)?

Reply:
okay, sounds good! I forgot that we're suggesting `iex` there